code/game/objects/items/devices/flashlight.dm

```diff
@@ -123,7 +123,7 @@
 	turn_on()
 
 /obj/item/flashlight/flare/New()
-	fuel = rand(800, 1000) // Sorry for changing this so much but I keep under-estimating how long X number of ticks last in seconds.
+	fuel = rand(290, 450) // Sorry for changing this so much but I keep under-estimating how long X number of ticks last in seconds.
```

Comment:
If the "fuel" var is how much ticks the flare is active, then the 450 value would be equivalent to 1,67 minutes. Ignore this comment if it's unrelated.